lib/prefab/logger_client.rb

```diff
@@ -107,10 +107,10 @@ def silence
 
     # Find the closest match to 'log_level.path' in config
     def level_of(path)
-      closest_log_level_match = @config_client.get(BASE) || :warn
+      closest_log_level_match = @config_client.get(BASE, :warn)
       path.split(SEP).inject([BASE]) do |memo, n|
         memo << n
-        val = @config_client.get(memo.join(SEP))
+        val = @config_client.get(memo.join(SEP), nil)
```

Comment:
We explicitly _do_ want a `nil` here because we're looking at deeper levels and don't want a default value to come into play.